sql/bduni_convert.sql

```diff
@@ -13,29 +13,33 @@ CREATE USER MAPPING FOR {user}
 
 GRANT USAGE ON FOREIGN SERVER bduni_server TO {user};
 
-DROP FOREIGN TABLE IF EXISTS troncon_de_route CASCADE;
-DROP FOREIGN TABLE IF EXISTS non_communication CASCADE;
-IMPORT FOREIGN SCHEMA public LIMIT TO (troncon_de_route, non_communication)
+DROP FOREIGN TABLE IF EXISTS {output_schema}.troncon_de_route CASCADE;
+DROP FOREIGN TABLE IF EXISTS {output_schema}.non_communication CASCADE;
+
+IMPORT FOREIGN SCHEMA {input_schema} LIMIT TO (troncon_de_route, non_communication)
 FROM SERVER bduni_server
-INTO public;
+INTO {output_schema};
+
+
+CREATE SCHEMA IF NOT EXISTS {output_schema};
```

Comment:
I think this one should arrive earlier. Because the previous command fail if output_schema doesn't exist. What do you think ?

Reply:
You're right ! I only tested when the schema was already available. I will update the .sql file.

Reply:
cool !